controllers/ironic_controller.go

```diff
@@ -668,24 +712,10 @@ func (r *IronicReconciler) generateServiceConfigMaps(
 
 	templateParameters := make(map[string]interface{})
 	if !instance.Spec.Standalone {
-
-		keystoneAPI, err := keystonev1.GetKeystoneAPI(ctx, h, instance.Namespace, map[string]string{})
-		if err != nil {
-			return err
-		}
-		keystoneInternalURL, err := keystoneAPI.GetEndpoint(endpoint.EndpointInternal)
-		if err != nil {
-			return err
-		}
-		keystonePublicURL, err := keystoneAPI.GetEndpoint(endpoint.EndpointPublic)
-		if err != nil {
-			return err
-		}
```

Comment:
Yeah, I'd be inclined to keep this if the aim is optimisation. We can always optimise later based on measurements.